apps/server/src/controllers/start-race.controller.ts

```diff
@@ -67,6 +69,15 @@ export const startRaceController = async ({
     protocol: socketProtocols.StartRaceAccept,
     data: startedRaceData,
   });
+
+  const raceEndTime = (race.timeoutDuration + RACE_END_WAIT_TIME) * 1000;
+
+  const raceTimeout = setTimeout(() => {
+    pubsub.publish(PubSubEvents.RaceTimout, {
```

Comment:
don't we have to check if the race has already started or not? before missing timeout

Reply:
@supunTE let's discuss this later.